ruma-common/Cargo.toml

```diff
@@ -12,6 +12,7 @@ edition = "2018"
 
 [dependencies]
 js_int = { version = "0.1.9", features = ["serde"] }
+ruma-api = { version = "=0.17.0-alpha.1", path = "../ruma-api" }
```

Comment:
This new dependency doesn't seem to actually be used.

Reply:
Ah, I'm not using `Outgoing` anymore.